docs/RELEASE_NOTES.md

```diff
@@ -4,7 +4,8 @@
 
 - Changes
   - MongoDB database of the statefulSet is managed using distinct Role, ServiceAccount and RoleBinding.
-
+  - TLS Secret can also contain a single "tls.pem" entry, containing the concatenation of the certificate and key
+    - If a TLS secret contains all three entries, the operator will raise an error if the tls.pem entry is not equal to the concatenation of the tls.crt and tls.key entries
```

Comment:
> If a TLS secret contains all three entries

this is a concatenation of `crt` and `key` right? so three ?

Reply:
I don't get the question😅 We both wrote "three"

Reply:
of sorry I mean `two` : -)

Reply:
I think three is correct: the error happens if the secret contains:

- tls.key
- tls.crt
- tls.pem

and pem is different from key+crt.

Do you think I should reword it to be more clear?

Reply:
ah got it 
> Do you think I should reword it to be more clear?

yes, please I think it's mentioned at multiple places , we can make it a bit more clearer

Reply:
will do, thanks!